lib/livebook_web/live/settings_live.ex

```diff
@@ -172,6 +195,23 @@ defmodule LivebookWeb.SettingsLive do
         />
       </.modal>
     <% end %>
+
+    <%= if @live_action in [:add_env_var, :edit_env_var] do %>
+      <.modal
+        id="env-var-modal"
+        show
+        class="w-full max-w-3xl"
+        js={JS.push("clear_env_var")}
```

Comment:
It's hard to tell where the `js` is evaluated, maybe `on_close`?

Reply:
Yeah, it works better tbh